### ISSUE TITLE
calculate the results on the test set

### ISSUE THREAD

**Issue by wangzai-qian** [chips96/DEYOLO#6]
Hello, I would like to calculate the results on the test set. I have changed `split` to `test`, but the results obtained by running the following demo are still from the validation set (val).

import warnings
warnings.filterwarnings('ignore')
from ultralytics import YOLO

if __name__ == '__main__':
    model = YOLO('best.pt')
    model.val(data='D:/DEYOLO/data.yaml',
              imgsz=640,
              batch=8,
              device='0',
              split='test',
              # iou=0.7,
              # rect=False,
              # save_json=True, # if you need to cal coco metrice
              project='runs/test',
              name='DEYOLO',
              )

**Comment by MrNeoBlue** [comment]
how you solve this issue? replacing the dir in the M3FD.yaml?
the result on test set is very low in my trained weights using deyolon. mAP50=38.7
